contents/basepages/aboutus/index.en.mdx

```diff
@@ -6,3 +6,162 @@ author: TSUTSUMI Kikuo
+
+### the beginning
+
+When I got married, I realized that I had no future as a company employee.
+I decided to start a side business of making handmade products.
+
+Since I was a child, I have been making things, drawing pictures, taking photographs, making small films, and performing on stage in theater.
+Because I love creating things.
+
+### You never know until you try
+
+<ImageBox alt="Suffering" float="left" image={2}
+	  style={{borderWidth: "1px"}} >
+<footnote>When I was worried about not selling</footnote>
+</ImageBox>
+
+However, to my surprise, it did not sell at all. In the first place, you can't even see the product page.
+
+I was very particular about it and spent a lot of time making it, but I was worried about why no one was interested.
+I was selfish and even felt resentment, thinking that if people could hold it in their hands, look at it, and use it, they would understand how good it is.
+
+But I can't afford to open a store.
+I had no choice but to sell it online.
+Now you can't even pick it up.
+
+### Try to change your mind
+- "A change in perspective is worth 80 IQ points." -- Alan Key (A father of personal computer.)<br/>
+  "A change of perspective is worth 80 points of IQ." -- Alan Kay, father of the personal computer
+
+So I started studying online marketing.
+I am studying business administration and software engineering at university.
+It's because I've been doing internet related work for a long time.
+
+Also, rather than trying various superficial methods, I thought that exploring fundamental and holistic methods would provide me with long-lasting knowledge.
+If you use a superficial method, when the situation changes, you have to study from scratch again...
+
+### Meet the right way for small shops
+
+Then, many small shops and companies cannot sell because
+I realized that it was because I imitated TV commercials and magazine advertisements for large companies.
+In other words, there was a method of advertising, publicity, and sales that was optimal for the size of the store and the size of the trade area.
+
+It's called DRM (Direct Response Marketing).
+Moreover, it is similar to the way of thinking and way of thinking that was done in traditional Japanese business.
+
+Do you know Drucker, who is famous for "Moshi Dora"?
+It is said that Echigoya in the Edo period started the world's oldest marketing.
+
+So, if you are Japanese, the content of my Google Maps attracting customers using DRM and DRM is
+It's all about the things that make sense honestly.
+
+However, when applying the Internet,
+Internet and computer knowledge are required.
+
+It's not easy for anyone to use.
+
+### Meet Google Maps, easy to use on your smartphone
+
+However, it is ideal for small local shops,
+Anyone can use it easily with a smartphone,
+And I found out about free Google business profiles.
+Google business profile
+It is a tool to manage the information of the shop placed on Google Maps.
```

Comment:
In MDX format files, multiple lines between empty lines and empty lines become one sentence. Please delete the line break when translating.

Reply:
done